dir.props

```diff
@@ -485,6 +485,11 @@
       </PropertyGroup>
     </Otherwise>
   </Choose>
+  
+  <!-- Default Test platform to deploy the netstandard compiled tests to -->
+  <PropertyGroup>
+    <TestTFM Condition="'$(TestTFM)'==''>netcoreapp1.0</TestTFM>
```

Comment:
That is the idea. The default for CI should be netcoreapp1.0. Currently testing this now.